### PR TITLE
Alignment

### PR DESCRIPTION
**Pull request by Raghavdata**

### CODE REVIEW

File: bin/alignment.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
+import pandas as pd
+import numpy as np
+from scipy.interpolate import interp1d
+from sys import stdin
+import sys
```
Comment (ramb0111) [inline bin/alignment.py]: according to python imports convention , the imports must be in order of first python built-in libraries , then 3rd party modules , at last project related modules

File: bin/alignment.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+import json
+import statsmodels.api as sm
+import warnings
+
+
+input_json = ''
```
Comment (ramb0111) [inline bin/alignment.py]: global variables should be avoided , why you are using it?

File: bin/alignment.py [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+def group_json_to_df(g, group_num, groups_data):
```
Comment (ramb0111) [inline bin/alignment.py]: No documentation for a any function

File: bin/alignment.py [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+
+def group_json_to_df(g, group_num, groups_data):
+    """This function converts the input group json into a pandas dataframe"""
+    return pd.DataFrame({'group': g,'sample': groups_data[g][group_num].keys(), 'rt': groups_data[g][group_num][groups_data[g][group_num].keys()[0]]})
+
+
+
+def groups_json_to_df(g, groups_data, minSample, extraPeaks):
+    """This function converts the input groups json into a pandas dataframe and subsets the data"""
```
Comment (sahil21) [inline bin/alignment.py]: Put proper doc strings. Explain input arguments and returns.

**Pull request closed without merging**